eclipsecs-sevntu-plugin/pom.xml

```diff
@@ -29,9 +29,9 @@
     <eclipsecs.version>10.4.0-SNAPSHOT</eclipsecs.version>
     <!-- verify time version -->
     <checkstyle.version>10.7.0</checkstyle.version>
-    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</checkstyle.configLocation>
+    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle-checks.xml</checkstyle.configLocation>
```

Comment:
This change has to be delayed. `checkstyle.version` is released version, not snapshot.

Reply:
Reverted. We can open an issue for this after this PR gets merged, yes?

Reply:
Yes we can.